.eslintrc.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    extends: './index.js'
-}
+    extends: './legacy.js'
+};
```

Comment:
why this?

Reply:
This is saying: When linting _this_ repository (`eslint-config-expensify`) we lint it with the ruleset of the "legacy" rules. And since we only write plain vanilla JavaScript, I figured that's alright. We don't need plugins like `eslint-plugin-react` or `eslint-plugin-jsx-a11y` here.